Makefile

```diff
@@ -155,13 +155,20 @@ endif
 endif
 endif
 
-# Enable libfido2 for buildbox and local environments that have it.
-ifneq ("$(shell ls {/opt/homebrew,/usr/lib/x86_64-linux-gnu,/usr/local/lib}/libfido2.* 2>/dev/null)","")
-LIBFIDO2_TAG := libfido2
-else
+# Enable libfido2 for testing?
+# Eargerly enable if we detect the package, we want to test as much as possible.
+ifeq ("$(shell pkg-config libfido2 2>/dev/null; echo $$?)", "0")
```

Comment:
Is `pkg-config` now required for local builds?

Reply:
Nope, we skip the if you don't have it. If you want to build go-libfido2, though, you'll need it.